ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/fixtures/cath121-body.json

```diff
@@ -1,16 +1,16 @@
 {
     "data": {
         "name": "Bristol gateway",
-        "trustName": "Bristol gateway",
-        "applicationId": "Cath121",
+        "trustName": "SOUTH MANCHESTER LEARNING TRUST",
```

Comment:
Because the data on staging is slightly different I expect this might fail when it runs there.
Please could you mind running these tests on staging (let me know if you need the command).